examples/online-shop/Config.toml

```diff
@@ -0,0 +1,5 @@
+USER="balUser"
+PASSWORD="balpass"
+HOST="localhost"
+PORT=1521
+DATABASE="ORCLCDB.localdomain"
```

Comment:
```suggestion
DATABASE="ORCLCDB.localdomain"

```

Reply:
changed in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/36/commits/fe7671fcf8c033cd93419840a549cb2dd8ad96d1